index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="This is an index page of my html web pages"
+    />
+    <meta name="author" content="Chanakya" />
+    <link rel="stylesheet" href="index.css" />
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,800;1,900&display=swap"
+      rel="stylesheet"
+    /> -->
+    <!-- <meta http-equiv="refresh" content="10" /> -->
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">aspect ratio</div>
+    <!-- <img src="/sun.jpg" alt="Sun daddy" class="imgContainer" /> -->
+    <!-- <main class="gridContainer">
+      <header class="header">Header</header>
+      <section class="section">Section</section>
+      <footer class="footer">Footer</footer>
+      <div class="footerNavigation">navigation</div>
+    </main> -->
+    <!-- <div class="container">
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+      <div class="item"></div>
+    </div> -->
+    <!-- <div class="container">
+      <img src="/123.jpg" alt="123456" width="100px" height="100px" />
+      <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li>Test</li>
+      </ul>
+    </div> -->
+    <!-- <iframe
+      src="/pic1.png"
+      frameborder="1"
+      name="navbar"
+      longdesc="this is my iframe example"
+      width="100%"
+      height="360vh"
+      scrolling="yes"
+      marginwidth="400px"
+      align="left"
+    ></iframe> -->
+    <!-- <hr />
+    <del>CHankaya</del>
+    <s>CHanakya</s> -->
+    <!-- <iframe src="/123.jpg" frameborder="1" name="first"> </iframe>
+    <iframe src="/345.jpg" frameborder="1" name="second"> </iframe>
+    <iframe src="/pic1.png" frameborder="1" name="third"></iframe>
+    <a href="/sun.jpg" target="third">Google</a> -->
+    <!-- <abbr title="World Health Organisation">WHO</abbr>
+    <h1>Chanakay</h1> -->
+    <!-- <div class="flag">
+      <div class="top"></div>
+      <div class="middle">
+        <div class="circle"></div>
+      </div>
+      <div class="bottom"></div>
+    </div> -->
+    <!-- <span class="loader">
+      <span class="ball"></span>
+      <span class="ball"></span>
+      <span class="ball"></span>
+    </span> -->
```

Comment:
same above

Reply:
we need to keep like only as per communication from PO